src/RestWrite.js

```diff
@@ -1665,6 +1665,16 @@ RestWrite.prototype._updateResponseWithData = function (response, data) {
       this.storage.fieldsChangedByTrigger.push(key);
     }
   }
+  for (const key in response) {
+    const skipKeys = ['objectId', 'createdAt', 'updatedAt', 'username'];
+    if (skipKeys.includes(key)) {
+      continue;
+    }
+    const value = response[key];
```

Comment:
this arbitrary list feels strange to me, could you explain why you need to skip these keys and more specifically "username" @dblythy  🙂

Reply:
These keys must always be returned by the API, even if they are unchanged (tests fail otherwise)

Reply:
Since they are essential keys, do we have them defined at some other place already, so we can reference that instead of building a new reference?

Reply:
I don't think they are exported anywhere else, there is `dafaultFields` in [LiveQuery/Client.js](https://github.com/parse-community/parse-server/blob/4c29d4d23b67e4abaf25803fe71cae47ce1b5957/src/LiveQuery/Client.js#L6)

Reply:
If you didn't find any other reference that we can make exportable and reference here, let's keep this new list. Maybe we stumble across something like this in the future.

Reply:
To be honest, i'm not sure. I think save responses should have these keys, _no matter_ if they are unchanged. There might be cases for other classes such as Installation where other keys are required, but I can't find any tests for them

Reply:
Could we open a security issue here? The list seems to be agnostic to class. What if a custom class has a `username` field and that field was not changed, will it be added (and thus involuntarily exposed) in the save response?